test/unittest/unittest_nntrainer_tensor.cpp

```diff
@@ -1163,7 +1361,7 @@ TEST(nntrainer_Tensor, constructor_from_shared_const_ptr_shares_variable_n) {
   nntrainer::Tensor B = *A;
   nntrainer::Tensor C = A->clone();
 
-  B.setValue(3, 4, 5, 6, 2.0f);
+  B.setValue(2, 3, 4, 5, 2.0f);
```

Comment:
any reason to update this here?

Reply:
It was wrong to update this index out of bound #532. Wonder how it managed to pass all the tests so far.